.github/actions/machinelaunchscript/action.yml

```
@@ -6,7 +6,8 @@ runs:
   steps:
     - run: |
         sudo yum -y remove git git224 git224-core ius-release.noarch # remove any older git versions and collateral first
+        sudo yum install -y libmpc
```

```suggestion
        sudo yum install -y libmpc # needed for build-toolchains.sh ec2fast precompiled toolchain in CI
```
Might be good to leave a comment about why we are installing mpc here.

---

Do you think i should move it into the docker image though? It's probably worth re-assessing what's needed here:
https://github.com/firesim/firesim/blob/main/.github/scripts/Dockerfile

---

It seems to me like most of it can be ripped out, since conda will bring in much of this.

---

Most of what @abejgonzalez is installing in the Dockerfile is trying to avoid really weird behavior that happens if you allow GH-A to clone the repo using the default centos7.6 version of git and then you're removing everything installed by the Dockerfile before machine-launch-script.sh gets run.

Although conda is going to pull in everything you want, I think you'll still end up hitting the weirdness Abe encountered with the GH-A checkout action using ancient git if you remove his workaround.

To me, it might make sense to change the Dockerfile so that it runs machine-launch-script.sh to create the docker image and have a new version get generated whenever machine-launch-script.sh changes?  That seems like it would be more work than I wanted to do right now though.

---

> I think i'd prefer to keep the container images as close to vanilla as possible, and then maybe rely on GH-A provided caching to speed up CI (and save cost).

Sure.  Sounds reasonable.

> We shouldn't need to upgrade pip anymore or add the the centos user though, right?

pip, defintiely not and I don't think we need to have centos user anymore but I'm not positive about that.